src/components/ui/EditForm/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../Button';
+import EditIcon from '../EditIcon';
+import { convertFromPixelsToRem, inputValidation } from '../../../misc';
+
+interface EditFormProps {
+  handleSubmit: () => void;
+  handleInputBlur: () => void;
+  loading?: boolean;
+  imageLink?: string;
+  displayHeader?: boolean;
+  imageName?: string;
+  submitButtonName: string;
+  children: (props: any) => React.ReactNode;
+}
+
+function EditForm(props: EditFormProps): React.ReactElement<EditFormProps> {
+  const [values, setValues] = React.useState({});
+  const [errors, setErrors] = React.useState({});
+  const {
+    handleSubmit = (): void => null,
+    handleInputBlur = (): void => null,
+    loading = false,
+    imageLink = '',
+    displayHeader = true,
+    imageName = '',
+    submitButtonName = '',
+    children,
+  } = props;
+
+  function handleChange(event): void {
+    const {
+      target: { value },
+    } = event;
+    setValues(prevState => ({ ...prevState, value }));
+  }
+
+  function handleBlur(event): void {
+    const {
+      target: { value },
+    } = event;
```

Comment:
There is no need to create the `value` variable if we are not using it now.

Reply:
Ok

Reply:
resolved